src/asSchemaField.jsx

```diff
@@ -206,6 +206,9 @@ const asSchemaField = (ComposedComponent, fieldType) => observer(class extends R
         onChange={this.onChangeValidate}
         onFocus={this.onFocus}
         onBlur={this.onBlur}
+        model={model}
```

Comment:
All the built-in widgets currently pass all props they don't recognize down to React-Toolbox, which in turn, does the same thing, so this will result in React errors that these props are not recognized in input tags, etc. Instead of these changes, I would recommend you use a custom field type and widget specified in the mapper. That widget component will receive the model, builder and mapper and can do what it needs to do (see Fieldset.jsx for an example). You could also use the asSchemaField HOC inside it as necessary.